stats/models/PlayerGeneralTraditionalTotal.py

```diff
@@ -23,7 +23,7 @@ class PlayerGeneralTraditionalTotal(Model):
     age = IntegerField(null=True)
     gp = IntegerField(null=True)
     w = IntegerField(null=True)
-    l = IntegerField(null=True)
+    l = IntegerField(null=True) # NOQA
```

Comment:
This is for flake8 to not complain.

Reply:
AH! Didn't realize there was a way to silence errors. Very nice.